pkg/export/storage.go

```diff
@@ -55,12 +55,12 @@ func NewStorage(exporter *Exporter) *Storage {
 
 // ApplyConfig applies the new configuration to the storage.
 func (s *Storage) ApplyConfig(cfg *config.Config) error {
-	return s.exporter.ApplyConfig(cfg)
+	return s.exporter.ApplyConfig(cfg, nil)
```

Comment:
question: Doesn't the rule evaluator need to refresh export options at runtime as well?